Library/Homebrew/rubocops/lines.rb

```diff
@@ -195,6 +195,41 @@ def autocorrect(node)
         end
       end
 
+      class ShellCmd < FormulaCop
+        def audit_formula(_node, _class_node, _parent_class_node, body_node)
+          popen_commands = [
+            :popen,
+            :popen_read,
+            :safe_popen_read,
+            :popen_write,
+            :safe_popen_write,
+          ]
+
+          find_every_method_call_by_name(body_node, :system).each do |method|
+            next unless match = regex_match_group(parameters(method).first, /[^"' ]*(?:\s[^"' ]*)+/)
```

Comment:
Making this regex a shared constant (with a comment) so it can be reused below would be nice.